apps/accounts/serializers.py

```diff
@@ -8,14 +8,23 @@
 
 # from apps.accounts.views import OTP_DummyToken
 
+
 # User registration
 class UserRegistrationSerializer(serializers.ModelSerializer):
     # We are writing this because  we need confirm password field in our Registration request
     password2 = serializers.CharField(style={"input_type": "password"}, write_only=True)
 
     class Meta:
         model = User
-        fields = ["email", "name", "password", "password2", "user_type"]
+        fields = [
+            "email",
+            "name",
+            "password",
+            "password2",
+            "user_type",
+            "age",
```

Comment:
age and gender we dont have to store in auth table

Reply:
okay, so I'll accept these fields in the request body, but will only save this in `tbl_user_profile`